protos/feast/core/Registry.proto

```diff
@@ -0,0 +1,35 @@
+//
+// * Copyright 2020 The Feast Authors
+// *
+// * Licensed under the Apache License, Version 2.0 (the "License");
+// * you may not use this file except in compliance with the License.
+// * You may obtain a copy of the License at
+// *
+// *     https://www.apache.org/licenses/LICENSE-2.0
+// *
+// * Unless required by applicable law or agreed to in writing, software
+// * distributed under the License is distributed on an "AS IS" BASIS,
+// * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// * See the License for the specific language governing permissions and
+// * limitations under the License.
+//
+
+syntax = "proto3";
+
+package feast.core;
+option java_package = "feast.proto.core";
+option java_outer_classname = "RegistryProto";
+option go_package = "github.com/feast-dev/feast/sdk/go/protos/feast/core";
+
+import "feast/core/Entity.proto";
+import "feast/core/FeatureTable.proto";
+import "google/protobuf/timestamp.proto";
+
+message Registry {
+    repeated Entity entities = 1;
+    repeated FeatureTable feature_tables = 2;
```

Comment:
Is the idea that we will first implement this registry and then come back to add FeatureViews?